parameter/ParameterMgr.cpp

```diff
@@ -102,6 +98,9 @@ using std::list;
 using std::vector;
 using std::ostringstream;
 
+// FIXME: integrate ParameterMgr to core namespace
```

Comment:
it will lead to unrelated changes, i will open a ticket and do it in an other pull request